src/pylibsshext/session.pyx

```diff
@@ -383,6 +393,19 @@ cdef class Session(object):
         if rc == libssh.SSH_AUTH_ERROR or rc == libssh.SSH_AUTH_DENIED:
             raise LibsshSessionException("Failed to authenticate with password: %s" % self._get_session_error_str())
 
+    def authenticate_interactive(self, password):
```

Comment:
add a PEP257 compliant docstring plz

Reply:
I'm not familiar with PEP257 to know if this passes, but this matches the style of the rest of the file.

Reply:
Unfortunately, most of the linters cannot check Cython so I just check some things myself...
FWIW here's the PEP, it's quite simple https://www.python.org/dev/peps/pep-0257/.
The idea of good docstrings is roughly the same as for the commit messages.
The first line should be a sentence (starting with a verb if it's a function/method) and there should be one empty line if you want to add an extra description. The title line is limited to 72 chars including the quotes. That's about it.

Those para/type things are not from the PEP, they are sphinx-specific and are currently unused but nice to have for humans.